<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="97,3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="rightMenuSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QSplitter" name="tableTurnSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>0</number>
       </property>
       <widget class="CustomMdiArea" name="tableArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="turnHeader" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>resource/TurnMenu/turn.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Gestion des tours</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>363</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="collapseButtonTurnMenu">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>resource/CollapsibleWidget/up.png</normaloff>
                <normalon>resource/CollapsibleWidget/down.png</normalon>resource/CollapsibleWidget/up.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="TurnMenu" name="turnWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="rightMenuWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="rightMenuLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QToolBox" name="actionMenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="TokenMenu" name="tokenPage">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>262</width>
               <height>326</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <attribute name="label">
              <string>Liste de jetons</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ChatWidget" name="m_ChatWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="footer" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="collapseButtonRightMenu">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>resource/CollapsibleWidget/sidebar.png</normaloff>resource/CollapsibleWidget/sidebar.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnexion">
    <property name="title">
     <string>Connexion</string>
    </property>
    <addaction name="actionConnection"/>
   </widget>
   <widget class="QMenu" name="menuPartie">
    <property name="title">
     <string>Partie</string>
    </property>
    <addaction name="actionCreateMap"/>
    <addaction name="actionOpenMap"/>
    <addaction name="separator"/>
    <addaction name="actionCreateImage"/>
   </widget>
   <addaction name="menuConnexion"/>
   <addaction name="menuPartie"/>
  </widget>
  <action name="actionCreateMap">
   <property name="text">
    <string>Cr√©er une carte</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>Connexion...</string>
   </property>
  </action>
  <action name="actionCreateImage">
   <property name="text">
    <string>Cr√©er une image</string>
   </property>
  </action>
  <action name="actionOpenMap">
   <property name="text">
    <string>Ouvrir une carte...</string>
   </property>
  </action>
  <action name="actionConnexion">
   <property name="text">
    <string>Connexion...</string>
   </property>
  </action>
  <action name="actionCr_er_une_carte">
   <property name="text">
    <string>Cr√©er une carte</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomMdiArea</class>
   <extends>QFrame</extends>
   <header>CustomMdiArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TokenMenu</class>
   <extends>QWidget</extends>
   <header>Token/TokenMenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header>Chat/ChatWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TurnMenu</class>
   <extends>QWidget</extends>
   <header>TurnMenu/TurnMenu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
